app/map-view.js

```diff
@@ -1,10 +1,11 @@
-/* global L */
+ /* global L */
 var fc = require('turf-featurecollection')
 var extent = require('turf-extent')
 var numeral = require('numeral')
 
 var accessToken = require('./mapbox-access-token')
 var polyColor = '#0571b0'
+var PolygonDataArray = []
```

Comment:
This is a little bit picky of me, but could we change the name `PolygonDataArray` and `NewPoly` so that the first letters are lowercase?  There's a strong convention in JS that names with the first letter capitalized are classes, whereas 'normal' functions and data variables start with a lowercase.  Because of that convention, it sorta messes with my head to see it this way :)